web3.md

```diff
@@ -587,5 +589,28 @@ WEB3 JSON RPC
     rule <k> #acctFromPrivateKey KEY => #newAccount #addrFromPrivateKey(KEY) ... </k>
          <accountKeys> M => M[#addrFromPrivateKey(KEY) <- #parseHexWord(KEY)] </accountKeys>
 
+    syntax KItem ::= "#firefly_addAccount"
+ // --------------------------------------
+    rule <k> #firefly_addAccount => #newAccount #parseHexWord( #getString("address", J) ) ~> #loadAccountData J ~> #sendResponse( "result": true ) ... </k>
+         <params> [ ({ _ } #as J), .JSONList ] </params>
+      requires isString( #getJSON("address", J) )
+
+    rule <k> #firefly_addAccount =>  #sendResponse( "error": {"code": -32025, "message":"Method 'firefly_addAccount' has invalid arguments"} ) ... </k> [owise]
+
+    syntax KItem ::= "#loadAccountData" JSON
+ // ----------------------------------------
+    rule <k> #loadAccountData J => . ... </k>
+         <account>
+           <acctID>  ACCTID  </acctID>
+           <balance>     ( _ => #if isString( #getJSON("balance",  J) ) #then #parseHexWord(#getString("balance", J)) #else 100                    #fi )     </balance>
+           <code>        ( _ => #if isString( #getJSON("code",     J) ) #then #parseByteStack(#getString("code",  J)) #else .ByteArray:AccountCode #fi )        </code>
+           <nonce>       ( _ => #if isString( #getJSON("nonce",    J) ) #then #parseHexWord(#getString("nonce",   J)) #else 0                      #fi )       </nonce>
+           <storage>     ( _ => #if isString( #getJSON("storage",  J) ) #then #parseMap(#getString("storage",     J)) #else .Map                   #fi )     </storage>
+           <origStorage> ( _ => #if isString( #getJSON("storage",  J) ) #then #parseMap(#getString("storage",     J)) #else .Map                   #fi ) </origStorage>
```

Comment:
Storage will be a `Map` not a `String`. So you just want to call `#getJSON` directly instead of `#getString`, and no need for the `isString`.